src/core/app.py

```diff
@@ -204,6 +218,7 @@ def _create_widgets(self):
             padx=5,
             pady=5
         )
+        tooltip.CreateToolTip(self.ent_stopped_message, constants.get('message'))
```

Comment:
```suggestion
        tooltip.CreateToolTip(self.ent_stopped_message, constants.get("message"))
```
Just changing single quotes to double quotes here.

Reply:
Don't forget to change this to double quotes.